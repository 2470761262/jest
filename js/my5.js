export default {
  generateConfig(){
    return {
      nickName:'谢谢你',
      nickName1:'忙着淋雨',
      nickName2:'sorry',
      time:new Date()
    }
  }
}