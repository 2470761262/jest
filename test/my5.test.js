import my5 from "../js/my5";

//快照测试
test("测试 generateConfig 函数", () => {
  expect(my5.generateConfig()).toMatchSnapshot({
    time: expect.any(Date)
  });
});

//行内快照，需要安装包
test("测试 generateConfig 函数", () => {
  expect(my5.generateConfig()).toMatchInlineSnapshot(
    {
      time: expect.any(Date) //可以写别的类型
    },
    `
    Object {
      "nickName": "谢谢你",
      "nickName1": "忙着淋雨",
      "nickName2": "sorry",
      "time": Any<Date>,
    }
  `
  );
});
