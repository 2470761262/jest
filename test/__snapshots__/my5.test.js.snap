// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateConfig 函数 1`] = `
Object {
  "nickName": "谢谢你",
  "nickName1": "忙着淋雨",
  "nickName2": "sorry",
  "time": Any<Date>,
}
`;
